### DISCUSSION TITLE
Update Profile Picture Rate Limit

### DISCUSSION THREAD

**Discussion by feri-irawan** [WhiskeySockets/Baileys#372] [General Questions] [answered]
I try to update my profile picture with another picture every 10 seconds. At first it worked fine, but after some time, I then got a `rate limit` error message

How much is the rate for changing your profile photo?

**Comment by FERDIZ-afk** [comment, marked as answer]
> I try to update my profile picture with another picture every 10 seconds. At first it worked fine, but after some time, I then got a `rate limit` error message
> 
> How much is the rate for changing your profile photo?

Be careful, friends, if WhatsApp detects what you are doing, it will definitely be banned and this repository will be closed.

**Reply by feri-irawan** [reply]
Really? Okay, I’ll stop here, I’m sorry, I appreciate the information. 🙏🏻
